pkg/catalog/routes_test.go

```diff
@@ -30,27 +30,39 @@ var _ = Describe("Catalog tests", func() {
 		})
 	})
 
-	Context("Test getTrafficPolicyPerRoute", func() {
-		It("lists traffic policies", func() {
-			allTrafficPolicies, err := getTrafficPolicyPerRoute(mc, tests.RoutePolicyMap, tests.BookbuyerService)
+	Context("Test getTrafficPoliciesForService", func() {
+		It("should return all the traffic policies associated with a service", func() {
```

Comment:
Can we have another case in this test whereby we see another route being added to the same src:dest traffic policy

Reply:
Done.